package.json

```diff
@@ -33,7 +33,7 @@
   "bin": "cli.js",
   "scripts": {
     "test": "make lint test",
-    "prepublish": "make dist"
+    "prepublish": "make NIS= dist"
```

Comment:
Cool.  Maybe add a comment in the Makefile.  `npm` won't be happy if there are comments in `package.json`.